### DISCUSSION TITLE
Taskbar windows is not correctly alligned to the taskbar (Windows 11 22H2 OS Build 22621.1413)

### DISCUSSION THREAD

**Discussion by fuadfajri** [zhongyang219/TrafficMonitor#1557] [Q&A]
As  mentioned in title, after Windows update, the traffic text on the taskbar seems not properly aligned to middle of the taskbar, it is just a little bit off to the upper edge of the taskbar. 

![Screenshot 2023-03-17 093946](https://user-images.githubusercontent.com/7019423/225798814-dfa9fa15-70e6-4b45-a541-66a6a3352b68.png)

Is there any workaround solution for this? Thank you.

**Comment by yanghaoynu** [comment]
me too ，i use  rainmeter plugin instead of it now， effect is okay，you can try it！
![Uploading Snipaste_2023-05-02_06-50-05.png…]()

**Comment by Lazy-Lad** [comment]
Same issue.
